### PR TITLE
Add Remove methods and more tests

### PR DESCRIPTION
**Pull request by JeremyKuhne**
Adds remove methods and starts adding more tests
Comment out ref handle override so the upstream obsolete can flow

@weshaggard, @ianhays 

Still have plenty more to do, pushing to keep the commit size down (for review purposes) and to fix the build break with Handle when updating packages

### CODE REVIEW

File: src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/IsolatedStorageFile.cs [hunk 1]
```diff
@@ -621,20 +659,82 @@ public override bool IncreaseQuotaTo(long newQuotaSize)
...
+                    return;
+                }
+            }
         }
 
         public static void Remove(IsolatedStorageScope scope)
         {
-            // TODO: https://github.com/dotnet/corefx/issues/11125
-            throw new NotImplementedException();
+            // The static Remove() deletes ALL IsoStores for the given scope
```
Comment (ianhays) [inline src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/IsolatedStorageFile.cs]: Weird behavior I recall you mentioning in the hall. Do the docs reinforce this?

File: src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs [hunk 2]
```diff
@@ -114,7 +114,7 @@ public partial class IsolatedStorageFileStream : System.IO.FileStream
         public override bool CanWrite { get { throw null; } }
         // https://github.com/dotnet/corefx/issues/11127
         // [System.ObsoleteAttribute("This property has been deprecated.  Please use IsolatedStorageFileStream's SafeFileHandle property instead.  http://go.microsoft.com/fwlink/?linkid=14202")]
-        public override System.IntPtr Handle { get { throw null; } }
+        // public override System.IntPtr Handle { get { throw null; } }
```
Comment (weshaggard) [inline src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs]: We would need to do this in the implementation as well.

Comment (weshaggard) [inline src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs]: Also can you please decouple this fix from the other changes as I want to get your fix in quickly to help unblock the package update.

File: src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/IsolatedStorageFile.cs [hunk 3]
```diff
@@ -533,24 +542,32 @@ private static IsolatedStorageFile GetStore(IsolatedStorageScope scope)
 
         public static IsolatedStorageFile GetStore(IsolatedStorageScope scope, Type applicationEvidenceType)
         {
+            // Scope MUST be Application
```
Comment (JeremyKuhne) [inline src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/IsolatedStorageFile.cs]: Left here for when we implement, if ever, as it isn't immediately obvious.

File: src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/IsolatedStorageFile.cs (hunk 1 above)
Comment (JeremyKuhne) [inline src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/IsolatedStorageFile.cs]: They do, but they're not the clearest in the world

**Comment by JeremyKuhne** [comment]
@dotnet-bot test Innerloop Windows_NT Debug Build and Test 

(System.Net.Tests.TaskFileWebRequestTestBase.ConcurrentReadWrite_ResponseBlocksThenGetsNullStream temp file in use)

File: src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs (hunk 2 above)
Comment (JeremyKuhne) [inline src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs]: The rest of the change is good to go, I've updated and will commit when green again.

**Comment by JeremyKuhne** [comment]
@dotnet-bot test Innerloop CentOS7.1 Release Build and Test
(System.Runtime.Tests exited with code 139. )

**Pull request merged**